assets/js/main.js

```diff
@@ -30,7 +30,7 @@ function initAutocomplete() {
         const startingWithLetter = remove(options, opt =>
           opt.match(new RegExp(`^${query}.+`, 'i')));
         const containingLetter = remove(options, opt =>
-          opt.match(new RegExp(`^.+${query}+`, 'i')));
+          opt.match(new RegExp(`^.*${query}*`, 'i')));
```

Comment:
if it's just a matter of "contains a letter" then it can be just `${query}` I guess.

Reply:
@barbaracassani So just this: `opt.match(new RegExp(`${query}`, 'i')`?

Reply:
I think so, yes. Try :)

Reply:
@barbaracassani it seems to work manually. But there's four tests that always fail for it for the Benefit Type

Reply:
Hmm, may it be that the tests are wrong? Anyway it's no biggie, feel free to leave it.